index.html

```diff
@@ -8216,6 +8216,95 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="suggestion">
+			<rdef>suggestion</rdef>
+			<div class="role-description">
+					<p>A suggestion contains a proposed change to content that can be approved or rejected.
+					For example, in an editing system that supports multiple authors, one author may suggest a change, and another author would be responsible
+					for accepting or rejecting the suggestion.
+					The suggested change must be present as child content, as an insertion, a deletion, or one of each.</p>
+					<p>When a suggestion is accepted, the suggestion role should be removed.</p>
```

Comment:
Does this mean the *role value* of `suggestion` goes away, but the element and its `insertion` and/or `deletion` child(ren) remain? In which case, what is the new role for the element formerly roled `suggestion`?

OR does it mean that the *element* which has the `suggestion` role goes away? If so, I assume the removal includes all the `insertion` and/or `deletion` children.

I'm guessing the text means the latter, but it would be good to clarify things. Also mind putting `code` tags around "suggestion" when "suggestion" refers to the role?

Reply:
I've reworded it, PTAL.

I think it's mostly important that "suggestion" goes away -- whether or not the element is kept, a new role is used, or change markup is kept is up to the author.